src/joint_state_listener.cpp

```diff
@@ -66,7 +70,7 @@ JointStateListener::JointStateListener(const std::shared_ptr<RobotStatePublisher
   // ignore_timestamp_ == true, joins_states messages are accepted, no matter their timestamp
   n_tilde.param("ignore_timestamp", ignore_timestamp_, false);
   // get the tf_prefix parameter from the closest namespace
-  publish_interval_ = ros::Duration(1.0/max(publish_freq, 1.0));
+  publish_interval_ = ros::Duration(1.0/std::max(publish_freq, 1.0));
```

Comment:
Nit: `#include <algorithm>` at the top of the file for `std::max`.

Reply:
Include <algorithm> in 0826b6aae67b321fc50cb44d50679a9b28247def